laravel/QUICKSTART.md

```diff
@@ -0,0 +1,37 @@
+# Quick Start
+
+## Prerequisite
```

Comment:
I think we can add `composer and npm` to the prerequisite.
Also, can we add the installation link in case these two are not already installed.